docs/DeveloperGuide.adoc

```diff
@@ -767,33 +814,301 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-{More to be added}
+[discrete]
+=== Use case: Email multiple persons
+
+*MSS*
+
+1.  User requests to list persons
+2.  AddressBook shows a list of persons
+3.  User selects persons to email
+4.  User enters keyboard shortcut to email
```

Comment:
Try to remember how to write use cases from the tutorial. You can refer to the LO on use cases.
It should be the interaction between user and system. Your use case should alternate between 'User' and 'System'.

Notice this in several of your use cases. Review and update them.

Lines 3 and 4 perhaps can be condensed into 1 line?